.github/ISSUE_TEMPLATE/story-issue-template.md

```diff
@@ -7,36 +7,50 @@ assignees: ''
 
 ---
 
-[
-The user story should have a reason to exist: what do I need as the user described in the summary?
-This part details any detail that could not be passed by the summary.
-]
+<!--
 
+User stories follow this format:
 
-### Acceptance Criteria
+As  __persona__, so that __reason__, I would like __request__.
```

Comment:
We could split hairs on this stuff all day. I think the last template I ran across had something approximate to this:
> As [role/persona], in order to [goal to accomplish], I want [feature/functionality].

Reply:
I'm good with updating to reflect the suggestion, especially as it seems more inclusive and accessible (mentally).

Reply:
@sknep updated